content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -5,130 +5,139 @@ Troubleshooting
+   :abbr:`IoT (Internet of Things)` Box will need to be re-flashed to update the image. (see
+   :doc:`Flashing the SD Card <flash_sdcard>`)
 
-If you are not in any of the cases listed above, make sure that the IoT Box
-has correctly started, by checking that a fixed green LED is showing next to
-the micro-USB port.
+If none of the cases listed above correct the issue, then make sure that the
+:abbr:`IoT (Internet of Things)` Box has correctly started, by checking that a fixed green LED is
+showing next to the power port.
 
-I've connected my IoT Box but it's not showing in my database
--------------------------------------------------------------
+IoT Box is connected but it's not showing in the database
+---------------------------------------------------------
 
-When you connect an IoT Box to a database, the IoT Box might restart, if
-that is the case, it might take up to one minute before appearing in your
-database. If after some time the IoT is still not showing, make sure that
-your database can be reached from the IoT Box and that your server doesn't
-use a multi-database environment.
+When an :abbr:`IoT (Internet of Things)` Box connects to a database, the
+:abbr:`IoT (Internet of Things)` Box might restart, if that is the case, it might take up to one
+minute before appearing in the database. If after some time the :abbr:`IoT (Internet of Things)` is
+still not showing, make sure that the database can be reached from the
+:abbr:`IoT (Internet of Things)` Box and that the server doesn't use a multi-database environment.
 
-My IoT Box is connected to my database, but cannot be reached
--------------------------------------------------------------
+The IoT Box is connected to the Odoo database, but cannot be reached
+--------------------------------------------------------------------
 
-Make sure that the IoT Box and the device running the browser are located
-on the same network as the IoT Box cannot be reached from outside the local
-network.
+Make sure that the :abbr:`IoT (Internet of Things)` Box and the device running the browser are
+located on the same network as the :abbr:`IoT (Internet of Things)` Box cannot be reached from
+outside the local network.
+
+The HTTPS certificate doesn't generate
+--------------------------------------
+
+In order to generate a HTTPS certificate a Odoo database subscription is necessary for the \
+:abbr:`IoT (Internet of Things)` Box. Connecting the :abbr:`IoT (Internet of Things)` Box prior to
+configuring a subscription for the database and :abbr:`IoT (Internet of Things)` Box with the
+Account Manager will result in an unsecured connection.
+
+In addition, a firewall can also prevent the HTTPS certificate from generating correctly. In this
+case, deactivate the firewall until the certificate is successfully generated. It should also be
+noted that certain devices, such as a router that has a firewall built-in to the software of the
+unit can prevent the HTTPS certificate from generating.
```

Comment:
It's a bit too shallow in my opinion. Even if you indeed cover the most common case.
Note that I am working on a more complete documentation on the matter with some code change on the IoT to give precise error codes, see:
https://github.com/odoo/documentation/pull/3818/files
( note that the error code mention in it is not merged on Odoo's code yet)

Maybe it would be better to add here a :seealso: to my documentation (when it will be merged), what do you think ?

Reply:
I think linking your new doc would be good. We'll need to push these changes first and then make the change on your PR when it gets pushed, otherwise we will have a merge conflict. I sent this PR to be reviewed by my teammates today.

Reply:
@lse-odoo There would be a build error as I can't reference a page that hasn't been published yet. We probably should publish this PR first and then PR 3818 and make the reference then so there isn't a build error.

Reply:
ok! I was just confused as "merge conflict" was not the appropriate term in this context. But I do argue with you, we will edit whatever part is necessary after if needed :)